pokedex/src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,70 @@
+import logo from "../../assets/pokemonlogo.svg";
+import pokeball from "../../assets/pokeball.png";
+import { Link, useNavigate } from "react-router-dom";
+import "./Header.css";
+import React, { useState } from "react";
+import { getPokemonFetch } from "../../utils/api";
+
+function Header({
+  pokemonList,
+  filteredPokemonList,
+  searchTerm,
+  setSearchTerm,
+}) {
+  const [selectedPokemon, setSelectedPokemon] = useState(null);
+  const navigate = useNavigate();
+
+  // const showPokemon = async (url) => {
+  //   const pokemonData = await getPokemonFetch();
+  //   setSelectedPokemon(pokemonData);
+  // };
+
+  return (
+    <div className="header">
+      <header>
+        <Link to="/">
+          <img alt="logo" className="header__logo" src={logo} />
+        </Link>
+        <nav></nav>
+        <Link to="/aboutme">
+          <img alt="about me logo" className="header__ball" src={pokeball} />
+        </Link>
+        <nav></nav>
+      </header>
+
+      <main>
+        <div className="search__container">
```

Comment:
* According to the checklist and criteria, you need to use `BEM` in your project. Please, correct the css selectors according to BEM